classes/controllers/modals/submissionMetadata/SubmissionMetadataHandler.inc.php

```diff
@@ -115,6 +119,24 @@ static function getUserAllowEditMetadata($submissionId, $currentUserId, $stageId
 
 		return false;
 	}
+
+	/**
+	 * Get whether the user is by default allowed to edit submission metadata
+	 * @param $journalId int
+	 * @param $currentUserId int
+	 * @return boolean true if the user is allowed to edit metadata by default
+	 */
+	private static function isUserDefaultAllowedEditMetadata($journalId, $currentUserId) {
```

Comment:
There's lots of scope for confusion with the permissions stuff. Can we change this (and the docblock description) to more closely match the logic that's expressed in the permissions set? Maybe something like `canUserAccessUnassignedSubmissions`?